README.md

```diff
@@ -225,3 +226,31 @@ For further configuration options (some have specific requirements about your en
 ### Support with a production on-premise (self-hosted) installation
 
 [Get in touch](https://wire.com/pricing/).
+
+## Troubleshooting
+
+There are multiple artifacts which combine to form a running wire-server deployment; these include:
+- docker images for each service
+- kubernetes configs for each deployment (from helm charts)
+- configuration maps for each deployment (from helm charts)
+
+
+If you wish to get some information regarding the code currently running on your cluster you can run the following:
+
+```
+./bin/deployment-info.sh <namespace> <deployment-name (e.g. brig)>
+```
+
+Example run:
+
+```
+./deployment-info.sh demo brig
```

Comment:
Reminder to chris to update this output with the new fields as soon as a build with the new tag format completes